test/CurveHelper.ts

```diff
@@ -0,0 +1,100 @@
+import { ethers } from "hardhat";
+import dotenv from "dotenv";
+import {
+    abiCurveFactory,
+    abi3PoolImplementation,
+} from "./ABIs";
+import {
+    addressCurveFactory,
+    address3CRV,
+    addressCurve3Pool,
+} from "./MainnetHelper";
+
+/**  Adds liquidity to a Metapool
+ * @param addressPool: address of the pool
+ * @param amountToken1: amount (lvUSD)
+ * @param amountToken2: amount (3crv)
+ * @param owner: signer
+ * @param r: instance: ContractContextTest
+ */
+async function fundMetapool (addressPool, [amountLvUSD, amount3CRV], owner, r) {
+    const token3CRV = r.external3CRV;
+    const lvUSD = r.lvUSD;
+    await token3CRV.approve(addressPool, ethers.constants.MaxUint256);
+    await lvUSD.approve(addressPool, ethers.constants.MaxUint256);
+    const pool = await getMetapool(addressPool, owner);
+    // coin[0] += 8, coin[1] +=7
+    await pool.add_liquidity([amountLvUSD, amount3CRV], 1, owner.address);
```

Comment:
Add a more realistic "expected" value instead of "1":
`expected = pool.calc_token_amount(amounts, True) * 0.99`